sys/dev/altera/jtag_uart/altera_jtag_uart_cons.c

```diff
@@ -69,6 +72,11 @@ int		aju_cons_jtag_present;
 u_int		aju_cons_jtag_missed;
```

Comment:
This driver is full of sadness with global state and no use of bus functions... not your problem to solve, but maybe we'll want/need something more mature for the DE10-Pro.